src/services/contactReport.service.js

```diff
@@ -0,0 +1,29 @@
+import { $db } from '@/services/firebase.service'
```

Comment:
You could add a function, that iterates of this function to generate the list of the last few days.
```suggestion
   > function that is called for generating the list (with date and name)
   > iterate
        > function that generates date
        > function that pulls list of date with user
```

Reply:
Postponed. For multiple days: #30